CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 4.7.0 - 2016-08-25
+
+* `beforeFileRead` and `afterFileRead` API events were added. This allows plugins to be notified
+   when gemini is about to read the file and said file is finished to be read.
+*  Declare `CAPTURE` and `END_TEST` events as deprecated. `UPDATE_RESULT` and `TEST_RESULT` were added instead them.
+*  Change flat reporter behavior on gemini update.
```

Comment:
Add ability to see from flat reporter what image was updated on `gemini update` command